test/stacks-test.js

```diff
@@ -0,0 +1,118 @@
+// A test to verify cards can be created and interacted
+var Application = require('spectron').Application;
+var electron = require('electron-prebuilt');
+var assert = require('assert');
+const path = require('path');
+var fs = require('fs');
+var Card = require('../app/Card.js');
+var Canvas = require('../app/Canvas.js');
+var Stack = require('../app/Stacks.js');
+
+var app = new Application({
```

Comment:
This declaration of `app` is masked by the declaration of another `app` on line 25. Therefore, this declaration can be removed.